gulpfile.js

```diff
@@ -94,10 +124,42 @@ gulp.task('build', function(cb) {
 			gulp.src(paths.src)
 				.pipe(gulp.dest(path.join(getBuildDir(browser), paths.dest)));
 		});
+
+		if (config[browser].manifest) {
+			del(path.join(getBuildDir(browser), config[browser].manifest));
```

Comment:
Does this need to be deleted? Naively I'd assume that gulp would happily overwrite it.
Also, it's async; it should return a promise.